pom.xml

```diff
@@ -564,10 +563,26 @@
                         <dependency>
                           <groupId>com.puppycrawl.tools</groupId>
                           <artifactId>checkstyle</artifactId>
-                          <version>6.19</version>
+                          <version>${checkstyle.version}</version>
                         </dependency>
                       </dependencies>
                     </plugin>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
I believe findbugs needs to be configured like checkstyle to depend on build-tools so it's only run once per build in a multi-module setup. According to https://gleclaire.github.io/findbugs-maven-plugin/examples/multi-module-config.html, it should be just like checkstyle, but we that article uses an extension instead of it a plugin dependency, so pick whichever works.

Reply:
this turns out to be a different issue, but follow up in https://github.com/confluentinc/common/pull/92